oclc.py

```diff
@@ -539,8 +540,8 @@ def book_info_to_metadata(self, subgraph, book_info):
 
         self.log.info("Processing edition %s: %r", oclc_id, titles)
         metadata = Metadata(self.source)
-        metadata.primary_identifier, new = Identifier.for_foreign_id(
-            self._db, oclc_id_type, oclc_id
+        metadata.primary_identifier, new = IdentifierData(
+            type=oclc_id_type, identifier=oclc_id
```

Comment:
Sticking an Identifier object here instead of an IdentifierData was a lazy hack which stopped working once I started seriously calling methods on this Metadata object.